stable/distribution/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # JFrog Distribution Chart Changelog
```

Comment:
can you replace with: All changes to this chart will be documented in this file.